test/src/org/exist/xquery/TransformTest.java

```diff
@@ -49,8 +49,54 @@ public void transform() throws XMLDBException {
                 "<p>End Template 1</p>" +
                 "</doc>");
     }
-    
-    
+
+    @Test
+    public void isSameNodeTest() throws XMLDBException {
```

Comment:
@dizzzz you are too fast, that PR one hour old. You should give @RemiKoutcherawy a bit more time ;-)